app/assets/javascripts/application.js.erb

```diff
@@ -67,6 +67,10 @@ var _Lobsters = Class.extend({
     Lobsters._showDownvoteWhyAt("comment", voterEl, function(k) {
       Lobsters.vote("comment", voterEl, -1, k); });
   },
+  foldComment: function(folderEl) {
```

Comment:
Could you rename this to `updateAriaLabel`? It doesn't fold and unfold the comment.